drizzlepac/haputils/diagnostic_json_harvester.py

```diff
@@ -82,9 +82,14 @@ def get_json_files(search_path=os.getcwd(), log_level=logutil.logging.INFO):
     for search_pattern in search_patterns:
```

Comment:
Please update docstring for search path as now the search will be performed one level deeper if no JSON files are found.

I also think the docstring for the entire routine should be updated to indicate this routine (unless it is changed) does not do a recursive search of the specified search path, but rather only searches the specified path and one level deeper, as deemed necessary.